tests/test_cli.py

```diff
@@ -136,34 +136,45 @@ def test_cli_custom_dimensions():
     assert result_valid.exit_code == 0
 
 
-def test_cli_custom_dimensions_empty():
-    """Check that CLI raises an error when specifying an empty directory ('empty')"""
+def test_cli_custom_dimensions_fails():
+    """Check that CLI raises an error when specifying a non-existing
+    directory ('foo')"""
 
     result_valid = runner.invoke(
         cli,
         [
             "validate-project",
-            str(TEST_DATA_DIR / "non-default_dimensions_failing"),
+            str(TEST_DATA_DIR / "non-default_dimensions_passing"),
```

Comment:
I think with the way the tests have evolved you might want to rename the two directories `non-default_dimensions_passing` and `non-default_dimensions_failing` as this test uses the first directory and is supposed to fail while the next one uses the second and is supposed to run. Maybe something more descriptive could be an idea like `non-default_dimensions_one_empty` or something like this.